tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -479,3 +479,36 @@ func certKeyExist(ctx context.Context, tlsCertFile, tlsKeyFile string) error {
 func errIsACLEnabled(stdErr string) bool {
 	return strings.Contains(stdErr, aclEnabledMatchStr)
 }
+
+func (ks *KopiaSnapshotter) GetRepositoryStatus(field string) string {
+
```

Comment:
nit: remove the extra line here. The linter will likely catch it. Same in the function below.

You can run `make lint` locally.